test/integration/tests/changeauth.sh

```diff
@@ -51,13 +51,13 @@ new_lockPasswd=newpswd
 
 tpm2_clear
 
-tpm2_changeauth -o $ownerPasswd -e $endorsePasswd -l $lockPasswd
+tpm2_changeauth -w $ownerPasswd -e $endorsePasswd -l $lockPasswd
```

Comment:
I was on the fence about this, but this is in the section "Hierarchy commands" which contains tpm2_createprimary that should be using the -o for output. At least sanity within the group.

Reply:
I was really puzzled on which toggle to use and `-o` was like 90% of the time used for output files. `-w` was free and close to the owner semantic so i went for it.

Not only it sanitize the "hierarchy" group but also allow a coherent usage of `-o` across all the tools.